app/javascript/components/forms/SurveyForm.vue

```diff
@@ -10,30 +24,56 @@ import Turbolinks from "turbolinks";
 import axios from "axios";
 import { setAxiosHeaders } from "../../helpers/axios-helpers";
 import "survey-vue/modern.css";
```

Comment:
I don't have a strong preference but if you prefer using single quotes in js part I think let's change it all to single quotes.

Reply:
Updated the components edited in this branch to use double quotes consistently